Formula/gcc-arm-none-eabi.rb

```diff
@@ -0,0 +1,48 @@
+class GccArmNoneEabi < Formula
+  desc "GNU Arm Embedded Toolchain"
+  homepage "https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm"
+  url "https://developer.arm.com/-/media/Files/downloads/gnu-rm/8-2019q3/RC1.1/gcc-arm-none-eabi-8-2019-q3-update-src.tar.bz2"
+  version "8-2019-q3-update"
+  sha256 "e8a8ddfec47601f2d83f1d80c0600f198476f227102121c8d6a5a781d0c2eeef"
+
+  env :std
+
+  depends_on "wget" => :build
```

Comment:
Resources shouldn't be downloaded by the build script behind our back.

Reply:
Hopefully fixed. I have to wait to find out if it builds with homebrews' versions of packages.